src/main/java/org/broadinstitute/consent/http/db/DarCollectionSummaryDAO.java

```diff
@@ -22,53 +22,76 @@ public interface DarCollectionSummaryDAO extends Transactional<DarCollectionSumm
   @RegisterBeanMapper(value = Election.class)
   @UseRowReducer(DarCollectionSummaryReducer.class)
   @SqlQuery("""
-      SELECT c.collection_id as dar_collection_id, c.dar_code, latest_dar.submission_date, latest_dar.reference_id as latest_dar_reference_id, latest_dar.parent_id as latest_dar_parent_id,
-      latest_dar.closeout_approving_so_id as latest_dar_closeout_approving_so_id,  latest_dar.closeout_so_approval_timestamp AS latest_dar_closeout_so_approval_timestamp,
-       u.display_name as researcher_name, i.institution_name, e.election_id, e.status, e.dataset_id, e.reference_id, v.voteid as v_vote_id, dd.dataset_id as dd_datasetid,
-        v.user_id as v_user_id, v.vote as v_vote, v.electionid as v_election_id, v.createdate as v_create_date, v.updatedate as v_update_date, v.type as v_type,
+      SELECT c.collection_id as dar_collection_id, c.dar_code,
+        latest_dar.submission_date, latest_dar.reference_id as latest_dar_reference_id,
+        latest_dar.parent_id as latest_dar_parent_id,
+        latest_dar.closeout_approving_so_id as latest_dar_closeout_approving_so_id,
+        latest_dar.closeout_so_approval_timestamp AS latest_dar_closeout_so_approval_timestamp,
+        researcher.display_name as researcher_name, i.institution_name,
+        e.election_id, e.status, e.dataset_id, e.reference_id,
+        v.voteid as v_vote_id, dd.dataset_id as dd_datasetid,
+        v.user_id as v_user_id, v.vote as v_vote, v.electionid as v_election_id,
+        v.createdate as v_create_date,v.updatedate as v_update_date, v.type as v_type,
         (regexp_replace(latest_dar.data #>> '{}', '\\\\u0000', '', 'g'))::jsonb ->> 'projectTitle' AS name,
         (regexp_replace(latest_dar.data #>> '{}', '\\\\u0000', '', 'g'))::jsonb ->> 'status' AS dar_status,
         (regexp_replace(latest_dar.data #>> '{}', '\\\\u0000', '', 'g'))::jsonb ->> 'closeoutSupplement' AS closeout,
         ARRAY_AGG(dar_all.reference_id) AS reference_ids
       FROM dar_collection c
-      INNER JOIN users u
-        ON u.user_id = c.create_user_id
+      -- DAR Collection Researcher join
+      INNER JOIN users researcher
+        ON researcher.user_id = c.create_user_id
       LEFT JOIN institution i
-        ON i.institution_id = u.institution_id
+        ON i.institution_id = researcher.institution_id
+      -- DAC User join
+      INNER JOIN users dacUser
+        ON dacUser.user_id = :currentUserId
+      INNER JOIN user_role ur
+        ON dacUser.user_id = ur.user_id AND ur.role_id = :roleId AND ur.dac_id IS NOT NULL
+      INNER JOIN dac dac
+        ON ur.dac_id = dac.dac_id
+      -- Datasets available to DAC
+      INNER JOIN dataset d
+        ON d.dac_id = dac.dac_id
+      -- Restrict DARs to the most recent submission per collection
       INNER JOIN (
         SELECT DISTINCT ON (collection_id) *
         FROM data_access_request
         WHERE submission_date IS NOT NULL
         AND (LOWER(data->>'status') != 'archived' OR data->>'status' IS NULL)
```

Comment:
Would an index on this help?

Reply:
Great question ... will dig into that

Reply:
We do have a test specifically for the case of ensuring that archived DARs are not in the result set: `testGetDarCollectionSummaryArchivedNotIncluded`. Removing the OR condition triggers a new set of test errors that make me hesitant to introduce a regression here.